components/Navigation.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <div class="flex ml-auto gap-10 m-2 mr-20 space-x-1 text-l">
+    <div
+      v-for="(page, i) in pages"
+      :key="i"
+      class="space-y-0.5"
+      @mouseover="page.hover = true"
+      @mouseleave="page.hover = false"
+    >
+      <!-- main button -->
+      <NuxtLink :to="page.parent.url">
+        <button
+          class="w-32 bg-blue-300 hover:bg-blue-500 text-white py-2 px-4 rounded"
+        >
+          {{ page.parent.title }}
+        </button>
+      </NuxtLink>
+      <!-- dropdown pages -->
+      <div
+        v-if="page.hover"
```

Comment:
Interesting read: https://learnvue.co/2021/05/v-if-vs-v-show-vue-conditional-rendering/#when-would-you-want-to-use-each-one